<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerPanel</class>
 <widget class="QWidget" name="customerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="scene">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1980</width>
     <height>1200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/background);</string>
   </property>
  </widget>
  <widget class="QLabel" name="customerLabel">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>20</y>
     <width>710</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addProductBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:green;
color:white;</string>
   </property>
   <property name="text">
    <string>افزودن کالا</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
color:white;</string>
   </property>
   <property name="text">
    <string>خروج</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="buyTable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>925</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="customerLabel_3">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>150</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>کل پرداختی</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="payLabel">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>150</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="customerLabel_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>کل فروش</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="buyLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>21</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1530</x>
     <y>100</y>
     <width>21</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="payTable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>240</y>
     <width>925</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="showProductBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:gray;
color:white;</string>
   </property>
   <property name="text">
    <string>کالاها</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
